style.css

```diff
@@ -0,0 +1,105 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;300;400;500&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-family:
+    'Poppins',
+    sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+}
+
+body {
+  max-width: 400px;
+  width: 100%;
+  min-height: 100vh;
+  background-color: #e5e5e5;
```

Comment:
- [ ] This `background-color` is different from the one in the template.  Please let's kindly correct it.  The correct background color shown in the Figma is `#FFFFFF` (plain white).

This is the main change that will give consistence to the colors on your project 😃